index.js

```diff
@@ -1,17 +1,57 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a,b)=> a-b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(this.items[pos] !== undefined){
+      return this.items[pos];
+    }else{
+      throw new Error('OutOfBounds');
+    }
+  }
 
-  max() {}
+  max() {
+    if(this.length === 0){
+      throw new Error('EmptySortedList');
+    }else{
+      var max = this.items.reduce(function(a, b) {
+        return Math.max(a, b);
+      });
+      return max;
+    }}
 
-  min() {}
+  min() {
+    if(this.length === 0){
+      throw new Error('EmptySortedList');
+    }else{
+      var min = this.items.reduce(function(a, b) {
+        return Math.min(a, b);
+      });
+      return min;
+    }}
 
-  sum() {}
+  sum() {
+    if(this.items.length === 0){
+      return 0;
+    }else{
+    return this.items.reduce((a, b)=> a+b);
+  }}
 
-  avg() {}
+  avg() {
+    if(this.items.length === 0){
+      throw new Error('EmptySortedList');
+    }else{
+      return this.items.reduce((a, b)=> a+b)/this.length;
```

Comment:
Well, once you already calculated the sum() above, you could just divide it by the length. :)